src/cadenza.js

```diff
@@ -417,14 +418,15 @@ export class CadenzaClient {
    */
   selectObjects(
     backgroundMapView,
-    { locationFinder, mapExtent, useMapSrs, signal } = {},
+    { locationFinder, mapExtent, useMapSrs, selectableLayers, signal } = {},
   ) {
     this.#log('CadenzaClient#selectObjects', backgroundMapView);
     const params = createParams({
       action: 'selectObjects',
       locationFinder,
       mapExtent,
       useMapSrs,
+      selectableLayers,
```

Comment:
Is this parameter already supported by the #show command?

Reply:
I think it's relevant only for the selectObjects action.